src/components/common/Tabs/TabItem.tsx

```diff
@@ -39,14 +32,10 @@ const Wrapper = styled.li<WrapperProps>`
   }
 `
 
-const TabItem: React.FC<TabProps> = (props) => {
-  const { onTabClick, id, title, isActive, activeColor } = props
-
-  return (
-    <Wrapper activeColor={activeColor} isActive={isActive} onClick={(): void => onTabClick(id)}>
-      {title}
-    </Wrapper>
-  )
-}
+const TabItem: React.FC<TabProps> = ({ onTabClick, title, activeColor }) => (
+  <Wrapper activeColor={activeColor} onClick={onTabClick}>
```

Comment:
this is the most important one. Look at how you made the fn in your changes and see what's different with mine and what that means for props etc

Reply:
I see:
- Destructuring props right from the argument passing:
- `isActive` not required to pass, given we can use `activeTab`. 
- onTabClick is already passed down as a prop, so no need to use an arrow fn with the `id` as an argument (already done in Tabs.tsx)

Reply:
yes last point is the important one